main.cpp

```diff
@@ -10,158 +10,162 @@ video: Chapter 2 - Part 3
-    return 0;    
+2) for each primitive type, write out 3 variable declarations inside the variableDeclaration() function on line 59.
+    a) give each variable declaration an initial value
+        - just ignore wchar_t. you do not need to declare 3 variables of type 'wchar_t'
+        - 'void' is a return type. you do not need to declare 3 variables of type 'void'.
+    b) at the end of the function, call ignoreUnused once and pass all of your variables to it. see line 71 for an example
+ 
+3) Declare 10 free functions
+    each declaration should have a random number of parameters in the function parameter list.
+    When naming your parameters, choose names that are relevant to the task implied by the function's name.
+    remember:
+    Name functions what they do
+        takeDogForWalk(int distanceInYards);
+    Name variables what they are.
+        int numStepsSinceStart;
+
+4) add { ignoreUnused( ); } after each declaration in place of the closing semicolon
+5) pass each of your function parameters to the ignoreUnused function like you did in b)
+6) if your function returns something other than void, add 'return { };' at the end of it.
+7) provide default values for an arbitrary number of parameters in the function parameter list.
+
+8) consult the coding style guide found in the Readme.MD and adjust the formatting of your 10 functions.  At this point, you might have something that looks like this:
+    float someFunc2(bool yes, int bar=2) { ignoreUnused(yes, bar ); return { }; }
+    This does not conform with the coding standard for this course (check the Readme.MD) and needs to be corrected
+    
+9) in the main function at the end: 
+    for each of those functions declared, 
+        a) write out how the function would look if called with correct arguments
+        b) if the function returned anything, store it in a local variable via the 'auto' keyword.
+        c) pass the local variables to ignoreUnused() as you did in 2b)
+    see main() for an example of this.
+ 
+10) click the [run] button.  Clear up any errors or warnings as best you can.
+ */
+
+//2)
+
+    void variableDeclarations()
+    {
+
+        int shoes = 33;
+        int violins = 40;
+        int ports = 2;
+
+        float height = 33.3f;
+        float weight = 94.8f;
+        float temperature = 98.5f;
+
+        bool areOpen = false;
+        bool canBeRemoved = true;
+        bool isVisible = false;
+
+        double bagels = 6004; 
+        double creamCheese = 884403;
+        double coffeeCups = 393939;
+
+        char grade = 'B';
+        char knob = 'H';
+        char view = 's';
+
+        unsigned int blackPants = 7;
+        unsigned int greenJackets = 5;
+        unsigned int shoulderPads = 567;
+
+        ignoreUnused(shoes, violins, ports, height, weight, temperature, areOpen, canBeRemoved, isVisible, bagels, creamCheese, coffeeCups, grade,         knob, view, blackPants,greenJackets, shoulderPads);
+
+    }
```

Comment:
-[ ] revise, all of this is indented to the right 4 more spaces than it should be.

Reply:
Corrected.